includes/import-products.php

```diff
@@ -162,7 +159,7 @@ function dw_format_and_save_dates(WC_Product $product, DateTime $date, array $co
 	wp_set_object_terms($product->get_id(), $date->format('H:i'), 'pa_tijd', false);
 	wp_set_object_terms($product->get_id(), date_i18n('l d F Y', $date->getTimestamp()), 'pa_pretty_date', false);
 
-	$i = 1;
+	$i         = 1;
```

Comment:
@Dirkhuethorst, what's up with the spaces here?

Reply:
Nothing